Src/AutoFixture.NUnit3.UnitTest/PilotTest.cs

```diff
@@ -0,0 +1,14 @@
+﻿using NUnit.Framework;
+
+namespace Ploeh.AutoFixture.NUnit3.UnitTest
+{
+    [TestFixture]
+    public class PilotTest
+    {
+        [Test]
+        public void AssertTrue()
+        {
+            Assert.IsTrue(true);
```

Comment:
That makes sense, but now that you've seen that it works, couldn't you remove it?

(I trust that unit tests will work - particularly if you tell me that you've seen it work. I don't need a dummy test to prove that.)